src/sidebars/star/star.html

```diff
@@ -1,2 +1,20 @@
 ﻿<img class="star" ng-src="assets/png/{{starCtrl.favouriteUrl()}}.png" ng-click="starCtrl.click(); $event.stopPropagation();"
      ng-mouseenter="starCtrl.mouseEnter()" ng-mouseleave="starCtrl.mouseLeave()" draggable="false" />
+
+<div class="confirmation-backdrop " ng-if="starCtrl.confirmationShow" ng-click="starCtrl.hideModal()">
+  <div class="confirmation-modal">
```

Comment:
Clicking on the modal itself closes it. Would a `$event.stopPropagation()` here fix this?